### PR TITLE
E0d/fix edx ansible

### PR DESCRIPTION
**Pull request by e0d**

### CODE REVIEW

File: playbooks/roles/edx_ansible/defaults/main.yml [hunk 1]
```diff
@@ -39,6 +39,10 @@ edx_ansible_venv_bin: "{{ edx_ansible_venv_dir }}/bin"
 edx_ansible_user: "edx-ansible"
 edx_ansible_source_repo: https://github.com/edx/configuration.git
 edx_ansible_requirements_file: "{{ edx_ansible_code_dir }}/requirements.txt"
+edx_ansible_requirements_files:
+  - "{{ edx_ansible_code_dir }}/pre-requirements.txt"
+  - "{{ edx_ansible_code_dir }}/requirements.txt"
+
 # edX configuration repo
-configuration_version: release
+configuration_version: "{{ EDX_CONFIGURATION_VERSION }}"
```
Comment (maxrothman) [inline playbooks/roles/edx_ansible/defaults/main.yml]: Is this defined anywhere else?

**Comment by maxrothman** [comment]
Nitpick: PR could use a more descriptive name. Other than that and the comment, :+1:

File: playbooks/roles/edx_ansible/defaults/main.yml (hunk 1 above)
Comment (e0d) [inline playbooks/roles/edx_ansible/defaults/main.yml]: in this PR, in group_vars/all

**Comment by singingwolfboy** [comment]
:+1:

**Pull request merged**